css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
+      box-shadow: rgb(64, 64, 64) 0px 0px 1px 0px inset !important
```

Comment:
You're right that that double nested nav shouldn't be there.
However, i just tried removing it in inspector and it makes no difference to styling. 

Besides which, it's the styling of the bottom element that we want, not the top one (which is kind of greyed out and bolded and thin-boxed)

The markup does look the same - I think the problem is that the "correct" look is happening through some accidental inheritance rather than being explicitly specified. 

I'm not saying we shouldn't remove the box-shadow, just saying let's not handle that issue as part of this PR.